wcfsetup/install/files/lib/http/error/ErrorDetail.class.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace wcf\http\error;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Provides various helper methods for PSR-7/PSR-15 request processing.
+ *
+ * @author  Tim Duesterhus
+ * @copyright   2001-2022 WoltLab GmbH
```

Comment:
2023. Also please get rid of these inconsistent whitespaces between the tag and the value. A single space is all that it takes.

Reply:
Fixed; That comment was outdated as well, copy and paste mistake.